packages/manager/modules/ng-layout-helpers/src/onboarding/translations/Messages_de_DE.json

```diff
@@ -0,0 +1,4 @@
+{
+  "pci_project_empty_more": "Mehr erfahren",
+  "pci_project_empty_new_tab": "Nouvel onglet"
```

Comment:
this key does not seem to be used

Reply:
yes, looks like it's used. It was there in cloud and I moved it as it is.

Reply:
This key has been removed since it is not used